src/get_started.md

```diff
@@ -12,11 +12,16 @@ index: 1
 
 Follow the steps below to try Rest.li quickly and get a basic idea of how it works:
 
-* [Build](#Build)
+* [Setup](#setup)
+* [Build](#build)
 * [Run the Examples](#run-the-examples)
 * [API](#api)
 * [Code Generator](#code-generator)
 
+## Setup
+
+JDK: You will need a Java 1.8 JDK to build the code. Using Java 1.9 or later will fail because Rest.li uses some APIs that were deprecated in 1.9 and the build treats warnings as error in order to ensure high code quality. On Linux systems openjdk-8-jdk is known to work.
```

Comment:
Unfortunately this is not entirely true. Certain version of openJDK 11  on linux can work as well. We mostly heard build failures only in MAC.

Maybe this should be one of the FAQ sections to suggest if build failed,try reverting back to java 1.8 :-)

Reply:
I think it would be useful to warn people up front that they likely need to use an older version of Java. My first try was with openjdk 14 on Linux and it failed. Is there a recommended version to use? Is it expected to build and work with newer versions like 14 (default on Ubuntu) or 17 (default when downloading from Oracle)?

Reply:
No problem, we can keep it here in this section for now. My point is just to say Java1.9's later version might still build.

Personally this is my java version, seems to work.

```
openjdk version "11.0.11" 2021-04-20 LTS
OpenJDK Runtime Environment 18.9 (build 11.0.11+9-LTS)
OpenJDK 64-Bit Server VM 18.9 (build 11.0.11+9-LTS, mixed mode, sharing
```